styles.css

```diff
@@ -0,0 +1,86 @@
+body {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  background: #539CED;
+}
+
+.main-elements-control {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  font-size: 20px;
+  color: white;
+  border: 3px solid #8976AD;
+  box-shadow: 10px 10px black;
+  width: 20%;
+  margin-bottom: 18px;
+}
+
+#content-page {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
```

Comment:
display: flex;
align-items: center;
justify-content: center;

these lines are often repeated. Try to either put them in one class and reuse it wherever required, or use nesting. Although I'm more for the first option, as the CSS should be minimally nested.